config.go

```diff
@@ -568,8 +568,8 @@ func loadConfig() (*config, []string, error) {
 		}
 
 		// Ensure pool fee is valid.
-		if cfg.PoolFee < 0 || cfg.PoolFee > 1 {
-			str := "%s: poolfee should be between 0 and 1"
+		if cfg.PoolFee < 0.002 || cfg.PoolFee > 0.05 {
+			str := "%s: poolfee should be between 0.002 and 0.05"
```

Comment:
Please include the corresponding percentage values of the bounds in the error messages as well.